Content.IntegrationTests/Tests/Damageable/AllSupportDamageableTest.cs

```diff
@@ -0,0 +1,121 @@
+using System.Linq;
+using System.Threading.Tasks;
+using Content.Shared.Damage;
+using Content.Shared.Damage.Components;
+using NUnit.Framework;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Map;
+using Robust.Shared.Prototypes;
+
+namespace Content.IntegrationTests.Tests.Damageable
+{
+    [TestFixture]
+    [TestOf(typeof(DamageableComponent))]
+    public class AllSupportDamageableTest : ContentIntegrationTest
```

Comment:
So these tests probably don't need a full integration test. You should be able to make a UnitTest with it instead. They run a lot faster because they don't spin up all the entity systems and managers.

Reply:
I'm running into the limits of my "code-monkey see, code-monkey do" approach here, and can't quite figure out how to do that just by reading other tests in Content.Tests. 

Is there a simple way to create a component-instance with a functional owner entity? Many of the functions in damageableComponent result in it raising events when the health gets changed, for which it tries to access the entity that owns it. If it weren't for that, I think I have a functional damageableComponent instance (although maybe created using egregious code).

Reply:
It's a RobustUnitTest, but https://github.com/space-wizards/RobustToolbox/blob/e16e0f4bd03c005954818beb9eb200609df3d5c4/Robust.UnitTesting/Shared/GameObjects/ContainerTests.cs has a test.

If you can't figure out a way to adapt it to a Unit Test then it's fine, but take a look and see if you can.